indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -77,6 +80,7 @@ protected void map(
       }
       catch (Exception e) {
         if (config.isIgnoreInvalidRows()) {
+          log.warn(e, "Ignoring invalid row [%s] due to parsing error", value.toString());
```

Comment:
this can potentially log a tremendous amount of errors. Can we cap the logging at something reasonable? Default == 10